deps/wazuh_testing/wazuh_testing/tools/monitoring.py

```diff
@@ -20,6 +20,7 @@
 import threading
 import time
 import yaml
+import sqlite3
```

Comment:
This import is unused.

Reply:
Done in 20cadf88fd2d3f4777f24434f6ca2b8e95ff0f11